sdk/azidentity/managed_identity_client.go

```diff
@@ -216,17 +212,7 @@ func (c *managedIdentityClient) createAccessToken(res *http.Response) (*azcore.A
 		if expiresOn, err := strconv.Atoi(v); err == nil {
 			return &azcore.AccessToken{Token: value.Token, ExpiresOn: time.Unix(int64(expiresOn), 0).UTC()}, nil
 		}
-		// this is the case when expires_on is a time string
```

Comment:
The format is now unified I take it?

Reply:
Yes, the former App Service version was the quirky one. All other platforms return Unix time as a string.